hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClientAbstract.java

```diff
@@ -3661,12 +3688,26 @@ private void doMultipartUpload(OzoneBucket bucket, String keyName, byte val,
     latestVersionLocations.getBlocksLatestVersionOnly()
         .forEach(omKeyLocationInfo ->
             assertNotEquals(-1, omKeyLocationInfo.getPartNumber()));
+
+    Map<String, String> keyMetadata = omKeyInfo.getMetadata();
+    assertNotNull(keyMetadata.get(ETAG));
+    if (customMetadata != null) {
+      for (Map.Entry<String, String> customEntry : customMetadata.entrySet()) {
+        assertTrue(keyMetadata.containsKey(customEntry.getKey()));
+        assertEquals(customEntry.getValue(), keyMetadata.get(customEntry.getKey()));
+      }
```

Comment:
Use `assertThat` instead of `assertTrue` for better message in case of failure.  It also provides assertions for maps.

```suggestion
      assertThat(keyMetadata).containsAllEntriesOf(customMetadata);
```

Reply:
Thank you for the suggestion. Updated with `assertThat` (also updated for other similar tests). 

I'll try to use `assertThat` API in the future.

Reply:
Some test failure due to the last commit, let me fix it first.

Reply:
Fixed.